websites/A/atomicradio/presence.ts

```diff
@@ -0,0 +1,64 @@
+const presence = new Presence({
+		clientId: "904084297831571518",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", () => {
+	const presenceData: PresenceData = {
+			largeImageKey: "https://i.imgur.com/16lVdWy.png",
+			smallImageKey: Assets.Search,
+			startTimestamp: browsingTimestamp,
+		},
+		player = document.querySelector<HTMLDivElement>("div.player");
+
+	if (player) {
+		const spaceId = player.querySelector(
+			"div.container > #spaceId"
+		).textContent;
+		presenceData.largeImageKey = player.querySelector(
+			"div.container > #artwork"
```

Comment:
Try to shorten selectors where possible like here:
```suggestion
			"#artwork"
```